tests/prague/eip7692_eof_v1/eip3540_eof_v1/test_jump_opcodes.py

```diff
@@ -0,0 +1,166 @@
+                    Section.Data("0x0bad60A7"),
+                ],
+            ),
+            "ef0001010004020001000704000400008000016001E100015B000bad60A7",
+            None,
+            id="rjumpi_valid",
+        ),
+        pytest.param(
+            # Check that jump tables work
+            Container(
+                name="EOF1V0013",
+                sections=[
+                    Section.Code(
+                        code=Op.PUSH1(1)
+                        + Op.RJUMPV[2, 0]
+                        + Op.ADDRESS
+                        + Op.POP
+                        + Op.ADDRESS
+                        + Op.POP
+                        + Op.STOP,
+                    ),
+                    Section.Data("0x0bad60A7"),
+                ],
+            ),
+            "ef0001010004020001000D04000400008000016001E2010002000030503050000bad60A7",
+            None,
+            id="jump_tables_valid",
+        ),
+        pytest.param(
+            # Check that jumps into the middle on an opcode are not allowed
+            Container(
+                name="EOF1I0019",
+                sections=[
+                    Section.Code(
+                        code=Op.PUSH1(1)
+                        + Op.RJUMPV[b"\x02\x00\x02\xFF\xFF"]
+                        + Op.ADDRESS
+                        + Op.POP
+                        + Op.ADDRESS
+                        + Op.POP
+                        + Op.STOP,
+                    ),
+                    Section.Data("0x0bad60A7"),
+                ],
+            ),
+            "ef0001010004020001000D04000400008000016001E2020002FFFF30503050000bad60A7",
+            EOFException.INVALID_RJUMP_DESTINATION,
+            id="rjump_invalid",
+        ),
+        pytest.param(
+            # TODO why here is expected an exception by the comment but test is valid
+            # Check that you can't get to the same opcode with two different stack heights
+            Container(
+                name="EOF1I0020",
+                sections=[
+                    Section.Code(
+                        code=Op.PUSH1(1) + Op.RJUMPI[1] + Op.ADDRESS + Op.NOOP + Op.STOP,
+                    ),
+                    Section.Data("0x0bad60A7"),
+                ],
+            ),
+            "ef0001010004020001000804000400008000016001E10001305B000bad60A7",
+            None,
+            id="jump_to_opcode_ok",
+        ),
+        pytest.param(
+            # Check that jumps into the middle on an opcode are not allowed
+            Container(
+                name="EOF1I0019",
+                sections=[
+                    Section.Code(code=Op.RJUMP[3] + Op.RJUMP[2] + Op.RJUMP[-6] + Op.STOP),
+                    Section.Data("0x0bad60A7"),
+                ],
+            ),
+            "ef0001010004020001000A0400040000800000E00003E00002E0FFFA000bad60A7",
+            EOFException.INVALID_RJUMP_DESTINATION,
+            id="rjump_3_2_m6_fails",
+        ),
+        pytest.param(
```

Comment:
Covered by: tests/prague/eip7692_eof_v1/eip4200_relative_jumps/test_rjumpi.py::test_rjumpi_into_rjumpi

Reply:
that test is jump to rjump, here is jump to opcode